src/components/LayerTree/LayerTree.js

```diff
@@ -93,9 +93,19 @@ const defaultProps = {
 class LayerTree extends Component {
   constructor(props) {
     super(props);
+    const { layerService, isItemHidden } = this.props;
+    const initialExpandedLayerNames =
```

Comment:
Couldn't we use the layer instances itself instead of their keys (`expandedLayers`)?

Reply:
Both are working. I do it with the layer instance instead.